InterruptClient.rb

```diff
@@ -0,0 +1,230 @@
+require 'socket'
+require 'json'
+
+require_relative './Console'
+
+class InterruptClient
+
+  CMD_QUIT = "\\"
+  START_MSG = 'Welcome! What is your name?'
+  INSTRUCTIONS = "Start typing to join the chat! To quit anytime, type #{CMD_QUIT}"
+  FAREWELL = "\r\nbye"
+  PROMPT = '> '
+
+  MAX_MSG_LENGTH = 3000 # max length of incoming message read
+  HANDSHAKE_WAIT = 2 # number of seconds to wait for ack from server before resending
+
+  COLORS = [
```

Comment:
If you're sure there won't be spaces in your color names I recommend `COLORS = %w{ green magenta etc }`